Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM alpine:3.8 as python 
 
 COPY requirements.txt /requirements.txt
+RUN sed -i 's/http\:\/\/dl-cdn.alpinelinux.org/https\:\/\/alpine.global.ssl.fastly.net/g' /etc/apk/repositories
```

Comment:
Change delimiter to some other character than slash (e.g. use ,). Then you don’t need to escape everything.